core/src/main/java/hivemall/evaluation/AUCUDAF.java

```diff
@@ -188,13 +234,19 @@ public void iterate(AggregationBuffer agg, Object[] parameters) throws HiveExcep
         public Object terminatePartial(AggregationBuffer agg) throws HiveException {
             ClassificationAUCAggregationBuffer myAggr = (ClassificationAUCAggregationBuffer) agg;
 
-            Object[] partialResult = new Object[6];
-            partialResult[0] = new DoubleWritable(myAggr.a);
-            partialResult[1] = new DoubleWritable(myAggr.scorePrev);
+            Object[] partialResult = new Object[11];
+            partialResult[0] = new DoubleWritable(myAggr.indexScore);
+            partialResult[1] = new DoubleWritable(myAggr.area);
             partialResult[2] = new LongWritable(myAggr.fp);
             partialResult[3] = new LongWritable(myAggr.tp);
             partialResult[4] = new LongWritable(myAggr.fpPrev);
             partialResult[5] = new LongWritable(myAggr.tpPrev);
+            partialResult[6] = myAggr.areaPartialMap;
```

Comment:
revise OI. `javaDoubleObjectInspector` instead of `writableDoubleObjectInspector`?